### PR TITLE
chore: Replace ImGui::InputText with components::input_text

### PR DESCRIPTION
**Pull request by TheGreenBandit**
completed #868

### CODE REVIEW

File: src/gui/components/input_text.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+	void components::input_text(const std::string_view label, char* buf, size_t buf_size, ImGuiInputTextFlags_ flag, std::function<void()> cb) {
+		if (ImGui::InputText(label.data(), buf, buf_size, flag))
+			if (cb)
+				g_fiber_pool->queue_job(std::move(cb));
+
+		if (ImGui::IsItemActive())
+			g_fiber_pool->queue_job([] {
+			PAD::DISABLE_ALL_CONTROL_ACTIONS(0);
+				});
```
Comment (LiamD-Flop) [inline src/gui/components/input_text.cpp]: Fix indentation

**Review by LiamD-Flop — CHANGES_REQUESTED** [review]
Remove the disable control from view_spoofing.cpp
https://github.com/YimMenu/YimMenu/blob/5e3af39854d6799ccb74e1d15b58536f9b7b8bfa/src/views/network/view_spoofing.cpp#L12-L14

File: src/core/globals.hpp [hunk 2]
```diff
@@ -168,10 +168,10 @@ namespace big
 			reaction request_control_event{ "Request Control Event", "Blocked Request Control Event from %s", "%s tried to mess with my vehicle!" };
 			reaction report{ "Report", "Blocked Report from %s", "%s tried to report me!" };
 
-			interloper_reaction breakup_others{ "Breakup Kicks On Other Players", "%s is trying to breakup kick %s!", "%s is trying to breakup kick %s!", true, true }; // blockable only when host but we have no way to specify that atm
```
Comment (maybegreat48) [inline src/core/globals.hpp]: This doesn't fix the issue, I will open a PR that actually fixes it soon

Comment (TheGreenBandit) [reply src/core/globals.hpp:171]: alr ill remove this part then

**Review by Yimura — APPROVED** [review]

**Pull request merged**